src/languages/en.js

```diff
@@ -136,6 +136,7 @@ export default {
         confirm: 'Confirm',
         splitBill: 'Split Bill',
         requestMoney: 'Request Money',
+        sendMoney: 'Send Money',
```

Comment:
Creepin' on your PR :D Did you mean to also add a translation in `es.js`?

Reply:
Was planning on adding all the translations in one big translation PR at the end!

Reply:
Sweet!